dependencies.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+sudo apt-get update
+sudo apt install -y build-essential
+sudo apt git
```

Comment:
1. delete last 2 lines
2. add empty newline to the end
3. add "&& sudo apt-get upgrade -y" after "sudo apt-get update"